verifier/verifier.go

```diff
@@ -153,33 +163,41 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 	}
 
 	// verify x509 trust store based authenticity
+	logger.Debug("Verify cert chain")
 	authenticityResult := verifyAuthenticity(ctx, trustPolicy, v.trustStore, outcome)
 	outcome.VerificationResults = append(outcome.VerificationResults, authenticityResult)
+	logVerificationResult(logger, authenticityResult)
 	if isCriticalFailure(authenticityResult) {
 		return authenticityResult.Error
 	}
 
 	// verify x509 trusted identity based authenticity (only if notation needs to perform this verification rather than a plugin)
 	if !slices.Contains(pluginCapabilities, proto.CapabilityTrustedIdentityVerifier) {
+		logger.Debug("Verify trust identity")
 		err = verifyX509TrustedIdentities(outcome.EnvelopeContent.SignerInfo.CertificateChain, trustPolicy)
 		if err != nil {
 			authenticityResult.Error = err
+			logVerificationResult(logger, authenticityResult)
 		}
 		if isCriticalFailure(authenticityResult) {
 			return authenticityResult.Error
 		}
 	}
 
 	// verify expiry
+	logger.Debug("Verify expiry")
 	expiryResult := verifyExpiry(outcome)
 	outcome.VerificationResults = append(outcome.VerificationResults, expiryResult)
+	logVerificationResult(logger, expiryResult)
 	if isCriticalFailure(expiryResult) {
 		return expiryResult.Error
 	}
 
 	// verify authentic timestamp
+	logger.Debug("Verify authentic timestamp")
```

Comment:
```suggestion
	logger.Debug("Validate authentic timestamp")
```

Reply:
updated to "Validating authentic timestamp"